src/Microsoft.ML.Data/Evaluators/EvaluatorUtils.cs

```diff
@@ -40,7 +40,7 @@ private static class DefaultEvaluatorTable
                         var tmp = new Dictionary<string, Func<IHostEnvironment, IMamlEvaluator>>
                         {
                             { AnnotationUtils.Const.ScoreColumnKind.BinaryClassification, env => new BinaryClassifierMamlEvaluator(env, new BinaryClassifierMamlEvaluator.Arguments()) },
-                            { AnnotationUtils.Const.ScoreColumnKind.MultiClassClassification, env => new MultiClassMamlEvaluator(env, new MultiClassMamlEvaluator.Arguments()) },
+                            { AnnotationUtils.Const.ScoreColumnKind.MultiClassClassification, env => new MultiClassClassificationMamlEvaluator(env, new MultiClassClassificationMamlEvaluator.Arguments()) },
```

Comment:
>MultiClassClassificationMamlEvaluator [](start = 105, length = 37)

Multiclass - note the casing

Reply:
Agree with Senja. A reference from wiki: [https://en.wikipedia.org/wiki/Multiclass_classification](https://en.wikipedia.org/wiki/Multiclass_classification) 

---
In reply to: [264526176](https://github.com/dotnet/machinelearning/pull/2919#discussion_r264526176) [](ancestors = 264526176)